diffkemp/simpll/Utils.cpp

```diff
@@ -257,10 +258,28 @@ bool isLogicalNot(const Instruction *Inst) {
     return false;
 }
 
+/// Returns true if the given instruction is a reorderable binary operation,
+/// i.e., it is commutative and associative.
```

Comment:
Let's add a note about floating-point operations not being associative to make sure we don't forget about this.